packages/cashier/src/stores/account-transfer-store.ts

```diff
@@ -438,9 +437,9 @@ export default class AccountTransferStore {
                 is_derivez: account.account_type === CFD_PLATFORMS.DERIVEZ,
                 ...(is_cfd && {
                     platform_icon:
-                        account.account_type === CFD_PLATFORMS.MT5 && combined_cfd_mt5_account
-                            ? combined_cfd_mt5_account.icon
-                            : cfd_icon_display,
+                        account.account_type === CFD_PLATFORMS.MT5 &&
```

Comment:
@thisyahlen-deriv the same thing here, we should leave the previous implementation
```
account.account_type === CFD_PLATFORMS.MT5 && combined_cfd_mt5_account
                            ? combined_cfd_mt5_account.icon
                            : cfd_icon_display,
```
because is_cfd flag includes not only MT5 acc, but also Dxtrade and DerivEZ accounts, so when is_cfd = true and we have Dxtrade or DerivEZ account `platform_icon` will be undefined and instead of proper icon we will show USD currency icon because account.is_mt = false => then we render Icon, but as account.platform_icon = undefined we will render the icon of the current currency (USD).
```
 return account.is_mt && account.platform_icon ? (
<img width="428" alt="Screenshot 2023-03-10 at 15 50 52" src="https://user-images.githubusercontent.com/103181646/224320534-ef4ed274-b636-4467-9396-153bae838780.png">

        <TradingPlatformIcon
            icon={account.platform_icon}
            size={size}
            className={appstore_icon_class_name}
            onClick={appstoreIconOnClickHandler}
        />
    ) : (
        <Icon
            icon={account.platform_icon || `IcCurrency-${account?.currency?.toLowerCase()}`}
            size={size}
            className={icon_class_name}
        />
    );
```